indra/llinventory/llparcel.cpp

```diff
@@ -166,16 +164,9 @@ void LLParcel::init(const LLUUID &owner_id,
 	mStatus = OS_NONE;
 	mCategory = C_NONE;
 	mAuthBuyerID.setNull();
-	//mBuyerID.setNull();
-	//mJoinNeighbors = 0x0;
-	mSaleTimerExpires.setTimerExpirySec(0);
-	mSaleTimerExpires.stop();
 	mGraceExtension = 0;
-	//mExpireAction = STEA_REVERT;
-	//mRecordTransaction = FALSE;
 
 	mAuctionID = 0;
-	mInEscrow = false;
```

Comment:
Notice: I'm not adding any code in this PR.  Everything is a delete: either class date members in this **ctor**, entire class methods, or in some places I only delete extra whitespace.